config.h.in

```diff
@@ -24,6 +24,6 @@
 #cmakedefine SYSCONFDIR "@SYSCONFDIR@"
 #cmakedefine SHAREDIR "@SHAREDIR@"
 
-#cmakedefine WITH_UNIT_TESTING 1
+#cmakedefine WITH_TESTING 1
```

Comment:
Make sure the Jenkinsfile would still run the tests.
Also make sure make test would still run the tests

Reply:
👍

Reply:
Now I'm wondering why we have UNIT_TESTING and WITH_TESTING

Reply:
It could be renamed, UNIT_TESTING is the cmake variable.